homeassistant/components/huum/strings.json

```
@@ -24,6 +24,11 @@
       "light": {
         "name": "[%key:component::light::title%]"
       }
+    },
+    "number": {
+      "humidity": {
+        "name": "Humidity"
```

Just as with "light" above we can use a reference here, too:

```suggestion
        "name": "[%key:component::sensor::entity_component::humidity::name%]"
```

This way it's automatically translated in all UI languages.

---

Thanks! Couldn’t find it.